.gitignore

```diff
@@ -1,4 +1,5 @@
 /*ide*
 /vendor
 /bin
-/src
```

Comment:
you should have your editor automatically add a newline at the end of the file on save.